pulp_ansible/app/galaxy/serializers.py

```diff
@@ -79,7 +79,7 @@ def get_versions_url(self, obj) -> str:
         return (
             "{hostname}/pulp_ansible/galaxy/{path}/api/v2/collections/{namespace}/{name}/"
             "versions/".format(
-                path=obj.path,
+                path=self.context["path"],
```

Comment:
Can you explain, why this should be better? To me it looks like the serializer would stop working if no context is given.

Reply:
Before we were annotating the object with the path every time. So for list endpoints we had to iterate over each object and set the path. So the serializer technically needed a context anyway since path wasn't apart of the object, this way is just more efficient and consistent with the other v3 views.

Reply:
Should we separate this improvement from the bugfix in case some things are requested for backports?